clang/test/dpct/cmake_migation/case_015/expected.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.24)
+project(AProject)
+cuda_add_library(target_one MODULE nosrc1.cu)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# No change unless property matches exactly
+set_target_properties(target_one
+                      PROPERTIES
+                      CXX_Standard 14)
+
+# No change unless value is one of 99, 11, 14
+# This negetive test will fail due to limitation in pattern-rewritter
+# set_target_properties(target_one
+#                       PROPERTIES
+#                       CXX_STANDARD 20)
+
+# Test CUDA_SEPARABLE_COMPILATION
+
+
+# No change unless property matches exactly
+set_target_properties(target_one
+                      PROPERTIES
+                      Cuda_SEPARABLE_COMPILATION Off)
```

Comment:
Cuda_SEPARABLE_COMPILATION is valid syntax in cmake? if not, we should not add this usage.

Reply:
`Cuda_SEPARABLE_COMPILATION` is not a CMake defined variable.